crawler/crawler/cloneRepository.go

```diff
@@ -28,22 +28,14 @@ func CloneRepository(domain Domain, hostname, name, gitURL, index string) error
 
 	// If folder already exists it will do a fetch instead of a clone.
 	if _, err := os.Stat(path); !os.IsNotExist(err) {
-		//	Command is: git fetch --all
 		out, err := exec.Command("git", "-C", path, "fetch", "--all").CombinedOutput() // nolint: gas
 		if err != nil {
 			return errors.New(fmt.Sprintf("cannot git pull the repository: %s: %s", err.Error(), out))
 		}
-		out, err = exec.Command("git", "-C", path, "reset", "--hard", "origin/HEAD").CombinedOutput() // nolint: gas
```

Comment:
Are we sure this is not needed anymore?

Reply:
Not 100% sure, but reset is not allowed in bare repositories.

Reply:
I think we are good, all the calculation we do for the vitality [start from HEAD](https://github.com/italia/developers-italia-backend/blob/e216011b724e8a8053fdddc401c0bcb8f9e213ff/crawler/crawler/repo_activity.go#L175) and use the previous commits.

HEAD is correctly set to the default branch in the bare clone.